server/app/api/v3.py

```diff
@@ -199,7 +199,7 @@ def register_api_v3(app, app_config, server_config, api_url_prefix):
 
     s3uri_api_path = "s3_uri"
     bp_s3uri = Blueprint(
-        f"api_dataset_{s3uri_api_path}_{api_version}",
+        f"api_dataset_{s3uri_api_path}_{api_version.replace('.',',')}",
```

Comment:
Is `.` a reserved character or something?

Reply:
Yea, in Flask 2 they became ineligible to use in the blueprint name -- something about [namespacing for nested blueprints](https://github.com/pallets/flask/issues/4041). As far as I could tell the blueprint name is just an internal handle that shows up in logs etc. but doesn't have any observable effect on the API server. Happy to be corrected if that's not true...